Bottom: 294ad5db1a94b17e67167884f4010c642055c7b1
Top:    199c14b7301f83f79b686d1d464572735fbe2db9
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-14 11:47:08 +0300

Refresh of ib-sock-into.patch

---

diff --git a/ib-sock.c b/ib-sock.c
index 8870570..1c67337 100644
--- a/ib-sock.c
+++ b/ib-sock.c
@@ -52,6 +52,7 @@ struct IB_SOCK *ib_socket_create()
 		goto out_free;
 	}
 
+	printk("IB socket create %p\n", sock);
 	return sock;
 out_free:
 	ib_socket_destroy(sock);
@@ -60,6 +61,7 @@ out_free:
 
 void ib_socket_destroy(struct IB_SOCK *sock)
 {
+	printk("IB socket destroy %p\n", sock);
 
 	if (sock->is_id)
 		rdma_destroy_id(sock->is_id);
